### ISSUE TITLE
WebView Crash

### ISSUE THREAD

**Issue by wbrawner** [wbrawner/SimpleMarkdown#7]
Logcat:
```
08-30 16:33:52.996 23045-23045/com.wbrawner.simplemarkdown E/ActivityThread: Activity com.wbrawner.simplemarkdown.view.activity.MainActivity has leaked IntentReceiver org.chromium.content.browser.accessibility.LollipopBrowserAccessibilityManager$1@9a08817 that was originally registered here. Are you missing a call to unregisterReceiver()?
                                                                             android.app.IntentReceiverLeaked: Activity com.wbrawner.simplemarkdown.view.activity.MainActivity has leaked IntentReceiver org.chromium.content.browser.accessibility.LollipopBrowserAccessibilityManager$1@9a08817 that was originally registered here. Are you missing a call to unregisterReceiver()?
                                                                                 at android.app.LoadedApk$ReceiverDispatcher.<init>(LoadedApk.java:1333)
                                                                                 at android.app.LoadedApk.getReceiverDispatcher(LoadedApk.java:1114)
                                                                                 at android.app.ContextImpl.registerReceiverInternal(ContextImpl.java:1405)
                                                                                 at android.app.ContextImpl.registerReceiver(ContextImpl.java:1378)
                                                                                 at android.app.ContextImpl.registerReceiver(ContextImpl.java:1366)
                                                                                 at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:603)
                                                                                 at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:603)
                                                                                 at org.chromium.content.browser.accessibility.LollipopBrowserAccessibilityManager.<init>(LollipopBrowserAccessibilityManager.java:3)
                                                                                 at org.chromium.content.browser.accessibility.BrowserAccessibilityManager.create(BrowserAccessibilityManager.java:1)
                                                                                 at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
                                                                                 at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:7)
                                                                                 at android.os.Handler.dispatchMessage(Handler.java:105)
                                                                                 at android.os.Looper.loop(Looper.java:164)
                                                                                 at android.app.ActivityThread.main(ActivityThread.java:6541)
                                                                                 at java.lang.reflect.Method.invoke(Native Method)
                                                                                 at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                                 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

**Comment by wbrawner** [comment]
Should be fixed by 414defaf479068c67710defe04fa605f959a6b10
